hack/ds.yaml

```diff
@@ -7,7 +7,7 @@ metadata:
     app.kubernetes.io/name: ds
     app.kubernetes.io/part-of: forgerock
 spec:
-  image: gcr.io/forgeops-public/ds-idrepo:7.1-dev
+  image: gcr.io/forgeops-public/ds-idrepo:7.1-pem
```

Comment:
you mean to commit this?

Reply:
We will need to coordinate the switch with PIT

Reply:
Just trying to work out what to do with making this change in forgeops.  Then I can keep the image the same

Reply:
reverted this